content-src/experiments/no-more-404s.yaml

```diff
@@ -89,3 +89,77 @@ eol_warning: The No More 404s experiment will not be removed from your browser a
+
+  <figure>
+  <img src="/static/images/experiments/no-more-404s/experiments_experimentdetail/2/4/24ddd4335aca6b96cca9106a6f3411d2_image_1470245154_0440.jpg">
+  <figcaption>We launched No More 404s with a restrained design.</figcaption>
+  </figure>
+
+  <p>Later on in the experiment we made a few changes to the add-on. First, we added in more common error codes associated with missing Web assets. Then we made a much larger change when we asked our design team to increase the visibility of the add-on. We wanted to see how these changes would affect user perception of the experiment.</p>
+
+  <figure>
+  <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" width="100%" height="auto" />
+  <figcaption>In December we changed to something a bit crazier.</figcaption>
+  </figure>
+
+  <p><strong>Here's what we learned</strong></p>
+
+  <p>Both teams wanted to get a sense of how many of the URLs that display 404 errors today had pages archived in the Wayback Machine. <a href="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/edit#gid=287298530" target="_blank">We tracked total 404s</a> encountered by Test Pilot participants (1,275,165) as well as results returned by the Archive (185,081). Over the course of the experiment, we averaged a result 14.5 percent of the time, but also observed that over the course of the experiment the daily average rose.</p>
+
+  <figure>
+  <iframe class="dashboard-embed" src="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/pubchart?oid=1805424694&amp;format=interactive" width=”600” height=”380”></iframe>
+  <figcaption>Both the total 404s encountered and the percentage of returned codes grew over the course of the experiment.</figcaption>  
+  </figure>
+
+  <p>In addition to 404s, which are common on the Web, there are other error codes users encounter. For example, users may reach a page with a 408 error code when a server takes too long to respond. In early November, the Internet Archive team decided to <a href="https://github.com/internetarchive/FirefoxNoMore404s/blob/master/src/scripts/background.js#L28" target="_blank">add more error codes to the experiment</a>. We hypothesized that adding additional error codes would cause the overall percentage of sites returned by the Wayback Machine to drop relative to total queries made by the experiment. However, the opposite was true: in the graph above there is a modest uptick in the percentage of valid returns in early November after we made the change.</p>
```

Comment:
nit: You may want to pin the GitHub link to /src/scripts/background.js#L28 above to a specific SHA (you can just press "Y" on that GitHub page and it will give you latest commit hash).

For example, https://github.com/internetarchive/wayback-machine-firefox/blob/9296b36df76752a0e01e57dee6f9ae40626512b6/src/scripts/background.js#L28

That way, if they refactor code or shuffle lines, it will always point to a the current state and not just highlight random lines (or take you to a 404 if they rename that page).